RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift

```diff
@@ -139,7 +139,7 @@ class APIWrappersViewController: ViewController {
 
         // also test two way binding
         let textValue = Variable("")
-        _ = textField.rx.textInput <-> textValue
+        (textField.rx.textInput <-> textValue).disposed(by: disposeBag)
```

Comment:
Hi @carlosypunto ,

We don't need `.disposed(by: disposeBag)` in this case and on line 164. I'm using this code to test will `<->` work without dispose bags.